skimage/feature/tests/test_corner.py

```diff
@@ -31,22 +32,32 @@ def im3d():
 def test_structure_tensor():
     square = np.zeros((5, 5))
     square[2, 2] = 1
-    Axx, Axy, Ayy = structure_tensor(square, sigma=0.1)
-    assert_array_equal(Axx, np.array([[ 0,  0,  0,  0,  0],
-                                      [ 0,  1,  0,  1,  0],
-                                      [ 0,  4,  0,  4,  0],
-                                      [ 0,  1,  0,  1,  0],
-                                      [ 0,  0,  0,  0,  0]]))
-    assert_array_equal(Axy, np.array([[ 0,  0,  0,  0,  0],
-                                      [ 0,  1,  0, -1,  0],
-                                      [ 0,  0,  0, -0,  0],
-                                      [ 0, -1, -0,  1,  0],
-                                      [ 0,  0,  0,  0,  0]]))
-    assert_array_equal(Ayy, np.array([[ 0,  0,  0,  0,  0],
-                                      [ 0,  1,  4,  1,  0],
-                                      [ 0,  0,  0,  0,  0],
-                                      [ 0,  1,  4,  1,  0],
-                                      [ 0,  0,  0,  0,  0]]))
+    Arr, Arc, Acc = structure_tensor(square, sigma=0.1, order='rc')
+    assert_array_equal(Acc, np.array([[0, 0, 0, 0, 0],
+                                      [0, 1, 0, 1, 0],
+                                      [0, 4, 0, 4, 0],
+                                      [0, 1, 0, 1, 0],
+                                      [0, 0, 0, 0, 0]]))
+    assert_array_equal(Arc, np.array([[0, 0, 0, 0, 0],
+                                      [0, 1, 0, -1, 0],
+                                      [0, 0, 0, -0, 0],
+                                      [0, -1, -0, 1, 0],
+                                      [0, 0, 0, 0, 0]]))
+    assert_array_equal(Arr, np.array([[0, 0, 0, 0, 0],
+                                      [0, 1, 4, 1, 0],
+                                      [0, 0, 0, 0, 0],
+                                      [0, 1, 4, 1, 0],
+                                      [0, 0, 0, 0, 0]]))
+
+
+def test_structure_tensor_orders():
+    square = np.zeros((5, 5))
+    square[2, 2] = 1
+    A_elems_default = structure_tensor(square, sigma=0.1)
```

Comment:
```suggestion
    with pytest.warns(FutureWarning,
                      match="The default order of the structure*"):
        A_elems_default = structure_tensor(square, sigma=0.1)
```

Emma, do you know if we switched to preferring this pytest-style over using the `expected_warnings` utility? I see both styles used in this file.

Reply:
We usually still use `skimage._shared.testing.expected_warnings`.

Reply:
Okay, I updated the suggestion to use `expected_warnings` instead of `pytest.warns`.

Reply:
Fixed, thanks!